### DISCUSSION TITLE
Is it possible to save the changes made from the python script before exporting?

### DISCUSSION THREAD

**Discussion by ssnuke** [spakin/SimpInkScr#73] [Q&A]
I need to import an image to an existing SVG file and edit the image (add rotation & scale image).

After adding the changes how can I export the SVG file with these changes in place, using the same script?

Currently when I try to export the file it exports the SVG file without any of the changes that has been done.

Is there any way to do this?

**Comment by spakin** [comment]
How have you been exporting the SVG file?  The file _should_ see all of the changes.
